editor.html

```diff
@@ -80,12 +80,14 @@ <h1>Flashcard Content Editor</h1>
       <div class="instructions">
         <h3>Instructions:</h3>
         <ol>
-          <li>Add your flashcard content using the table below</li>
-          <li>Click "Generate JSON" to create your flashcard data file</li>
+          <li>Add your flashcard content using the table below.</li>
+          <li>Click "Generate JSON" to create your flashcard data file.</li>
           <li>
-            Save the generated file in the same folder as your flashcard viewer
+            Save the generated file to your desired location (e.g., your Downloads folder).
+          </li>
+          <li>
+            You can name the file "flashcards.json" or any other descriptive name.
```

Comment:
There's [default behavior](https://github.com/roshan-sama/flash-card-game/blob/244d98e8a99d0180c3ecf9401886aab9f690f995/flashcard.html#L481) where if a user places a file called flashcards.json at the same level as the flashcard.html file, it gets picked up by default. Helpful if you want to practice those over and over without needing to upload a file each time.

I might add behavior description in in later

Reply:
Worth noting that I think this actually only works on a deployed server, so probably not useful for regular users that just have this downloaded on their local machine. Security rules prevent fetch from accessing file system contents.

On windows you can turn on IIS and have the contents of that folder be served as a webpage, on Mac maybe you can download docker and build it, but thats way too complex

Reply:
Hm, that's weird - if http-server is like Nginx it should serve all of the assets, including /flashcards.json, I might have to play around with making the path relative maybe for local deployments

This repo is deployed here: https://roshan-sama.github.io/flash-card-game/ via the default github pages deployment. Upload/download works fine there like you mentioned